### PR TITLE
Feature/sender dependent relayhost auth

### PR DESCRIPTION
**Pull request by smeagol74**
Please check it out. #21 should be allowed by this. At least it's working for me.

### CODE REVIEW

File: s6/postfix/run [hunk 1]
```diff
@@ -91,17 +91,37 @@ if [ "${RELAYHOST_AUTH}" == "yes" ]; then
...
+        if [[ "${#MAP[@]}" -eq 2 ]]; then
+          HOST=${MAP[0]}
+          USER=${MAP[0]}
+          PASS=${MAP[1]}
+        elif [[ "${#MAP[@]}" -eq 3 ]]; then
           HOST=${MAP[0]}
           USER=${MAP[1]}
           PASS=${MAP[2]}
-        elif [[ "${#MAP[@]}" -eq 4 ]]; then
+        elif [[ "${#MP[@]}" -eq 4 ]]; then
```
Comment (trnubo) [inline s6/postfix/run:124]: Is this a typo? (I haven't actually tested this yet)

**Comment by trnubo** [comment]
Thank you for this PR. Other than the one comment this looks good. I'll try to do some testing on this asap.

File: s6/postfix/run (hunk 1 above)
Comment (smeagol74) [reply s6/postfix/run]: Yes. This is typo. I've fixed it in following commit.

**Review by denisvic — APPROVED** [review]

**Comment by macropin** [comment]
As above this has been merged. Thanks for the PR.

**Pull request closed without merging**